scripts/prereqs/install_prereqs.sh

```diff
@@ -257,7 +263,7 @@ if [[ $BUILD_DISTRIBUTABLE_LIBRARY == false && $CENTOS_VERSION -eq 6 ]]; then
 fi
 
 RC=1
-install_prerequisites $2 &&
+install_prerequisites $2 $3 &&
```

Comment:
Can you document what $3 is? See lines 64 and 65.

Reply:
Done